dev/com.ibm.ws.security.wim.adapter.ldap_fat/build.gradle

```
@@ -138,6 +138,20 @@ autoFVT.doLast {
       into new File(autoFvtDir, 'publish/servers/' + server + '/dropins')
     }
   }
+  
+  /*
+   * Copy the truststore.p12 into test servers.
+   */
+  servers = [
+    'com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl',
+    'com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl'
+  ]
+  servers.each { server ->
+    copy { 
+      from project(':com.ibm.ws.security.wim.adapter.ldap_fat').file('resources/truststore.p12')
+      into file('publish/servers/' + server)
```

Not sure why you didn't copy into fvt dir like other examples.

```suggestion
      into new File(autoFvtDir, 'publish/servers/' + server
```

---

Usually we put this in `<server>/resources/security` probably should do so here, but that will require change to your server.xml files for this test (to point at the new dir).